man/systemd.resource-control.xml

```diff
@@ -474,6 +474,113 @@
+          socket-activated services, this configuration setting and the collected accounting data of the service unit
+          and the socket unit are kept separate, and displayed separately, no propagation of the setting and the
+          collected data is done, in neither direction. Moreover, any traffic sent or received on any of the socket
+          unit's sockets is accounted to the socket unit — and never to the service unit it might have activated, even
+          if the socket is used by it. The system default for this setting may be controlled with
+          <varname>DefaultIOAccounting=</varname> in
+          <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IPAddressAllow=<replaceable>ADDDRESS[/PREFIXLENGTH]</replaceable></varname></term>
+        <term><varname>IPAddressDeny=<replaceable>ADDRESS[/PREFIXLENGTH]</replaceable></varname></term>
+
+        <listitem>
+          <para>Turn on address range network traffic filtering for packets sent and received over AF_INET and AF_INET6
+          sockets.  Either of these directives take a space separated list of IPv4 or IPv6 addresses, each optionally
+          suffixed with an address prefix length (separated by a <literal>/</literal> character). If the latter is
+          omitted, the address is considered a host address, i.e. the prefix covers the whole address (32 for IPv4, 128
+          for IPv6).
+          </para>
+
+          <para>The access lists configured with this option are applied to all sockets created by processes of this
+          unit (or in the case of socket units, associated with it). The lists are implicitly combined with any
+          system-wide lists configured with the equally named global options in
+          <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>. By
+          default all access lists are empty. When configured the lists are enforced as follows:</para>
+
+          <itemizedlist>
+            <listitem><para>Access will be granted in case its destination/source address matches any entry in the
+            <varname>IPAddressAllow=</varname> setting.  </para></listitem>
+
+            <listitem><para>Otherwise, access will be denied in case its destination/source address matches any entry
+            in the <varname>IPAddressDeny=</varname> setting.</para></listitem>
+
+            <listitem><para>Otherwise, access will be granted.</para></listitem>
+          </itemizedlist>
+
+          <para>In order to implement a whitelisting IP firewall, it is recommended to use a system-wide
+          <varname>IPAddressDeny=</varname><constant>any</constant> setting, plus individual per-unit
+          <varname>IPAddressAllow=</varname> lines permitting network access to relevant services, and only them.</para>
+
```

Comment:
In other places the user is warned about kernel feature availability:
`Note that the implementation of this setting might be impossible (for example if user namespaces are not available), and the unit should be written in a way that does not solely rely on this setting for security.`

Reply:
will add such a comment